app/src/main/java/org/metabrainz/mobile/fragment/ArtistBioFragment.java

```diff
@@ -58,12 +58,50 @@ public void onActivityCreated(Bundle savedInstanceState) {
     }
 
     private void findViews(View layout) {
-        bioPicture = layout.findViewById(R.id.bio_picture);
-        yearsActive = layout.findViewById(R.id.years_active);
-        bioText = layout.findViewById(R.id.bio_text);
-    }
-
-    @Override
+        artistType = layout.findViewById(R.id.artist_type);
+        artistGender = layout.findViewById(R.id.artist_gender);
+        artistArea = layout.findViewById(R.id.artist_area);
+        artistLifeSpan = layout.findViewById(R.id.life_span);
+    }
+
+    private void getArtistWiki(){
```

Comment:
I think I read something on IRC chatlog about how this works, but I don't exactly remember. Could you add some comments about what are the steps to show Wikipedia/Wikidata information? (I think it's a good habit to be verbose at the beginning when the project is still being built)

Reply:
Yeah, sure. This function is in the works yet. I'll document it as soon as its over.